frontend/components/Bridge/BridgeOnEvm/DepositForBridging.tsx

```diff
@@ -149,7 +157,10 @@ export const DepositForBridging = ({
     canPollForBridgeRefillRequirements,
   );
 
-  // fetch bridge refill requirements manually on mount
+  /**
+   * fetch bridge refill requirements manually on mount, this is to ensure
+   * that stale values aren't shown - in case a user visits the bridging page again
+   */
```

Comment:
Note:
The difference between `//` and `/** ... */` is that `//` is used for regular inline comments meant only for developers, whereas `/** ... */` is used for documentation (JSDoc). When you hover over a function, variable, etc., the `/** ... */` comment is shown as a tooltip in the editor, while // comments are not.

```suggestion
   // fetch bridge refill requirements manually on mount, this is to ensure
   // that stale values aren't shown - in case a user visits the bridging page again
```